src/Uno.UI/UI/Xaml/Media/Brush.Android.cs

```diff
@@ -90,33 +90,18 @@ internal static IDisposable AssignAndObserveBrush(Brush b, ColorSetterHandler co
 			}
 			else if (b is ImageBrush imageBrush && imageBrushCallback != null)
 			{
-				var disposables = new CompositeDisposable(5);
+				var disposables = new CompositeDisposable(6);
 				imageBrushCallback();
+				 
+				void ImageChanged()
+				{
+					imageBrushCallback();
+				}
 
-				imageBrush.RegisterDisposablePropertyChangedCallback(
-					ImageBrush.ImageSourceProperty,
-					(_, __) => imageBrushCallback()
-				).DisposeWith(disposables);
-
-				imageBrush.RegisterDisposablePropertyChangedCallback(
-					ImageBrush.StretchProperty,
-					(_, __) => imageBrushCallback()
-				).DisposeWith(disposables);
-
-				imageBrush.RegisterDisposablePropertyChangedCallback(
-					ImageBrush.AlignmentXProperty,
-					(_, __) => imageBrushCallback()
-				).DisposeWith(disposables);
-
-				imageBrush.RegisterDisposablePropertyChangedCallback(
-					ImageBrush.AlignmentYProperty,
-					(_, __) => imageBrushCallback()
-				).DisposeWith(disposables);
+				imageBrush.ImageChanged += ImageChanged;
```

Comment:
These changes overlap the contents of https://github.com/unoplatform/uno/pull/12234 by @MartinZikmund, where we want to remove subscriptions and events in general (they're more costly than using method overrides). What do you think, @MartinZikmund?